mscore/preferences.cpp

```diff
@@ -113,6 +113,7 @@ void Preferences::init(bool storeInMemoryOnly)
             {PREF_IMPORT_MUSICXML_IMPORTLAYOUT,                    new BoolPreference(true, false)},
             {PREF_IMPORT_OVERTURE_CHARSET,                         new StringPreference("GBK", false)},
             {PREF_IMPORT_STYLE_STYLEFILE,                          new StringPreference("", false)},
+            {PREF_IMPORT_RESET_OLD_SCORES,                         new StringPreference("", false)},
```

Comment:
Shouldn't the default value here be `Ask`?

Reply:
None or "Ask" are treated the same

Reply:
I was thinking of makiing it a boolean setting, Yes/No, and the absence of the setting to mean "Ask", but the `Preferenmces::remove()` method is private.